docs/index.rst

```diff
@@ -43,6 +44,11 @@ From project's home page:
 
 The psutil documentation you're reading is distributed as a single HTML page.
 
+Installation
+============
+
+.. include:: ../INSTALL.rst
+
```

Comment:
I'm -1 on this because adds too many text to the doc which I prefer it remains an "API only doc". 
Instead you can simply mention the INSTALL instruction in the "quick links" section as a link.